docs/modules/ROOT/pages/licensing.adoc

```diff
@@ -0,0 +1,106 @@
+= Licensing
+
+[abstract]
+CMOS is composed of primarily other OSS software, not linked but used directly. This page provides an overview of the relevant licensing.
+
+== Background
+
+Before diving into this page, it is worth reviewing industry and standards bodies information on container licensing.
+A good place to start is with the link:https://www.linuxfoundation.org/tools/docker-containers-what-are-the-open-source-licensing-considerations/[Linux Foundation blog post].
+
+== Components
+
+This software uses the following components with their associated licensing also captured:
+
+* Grafana: AGPL 3.0 https://github.com/grafana/grafana/blob/main/LICENSE
+* Loki: AGPL 3.0 https://github.com/grafana/loki/blob/main/LICENSE
+* Prometheus: Apache 2.0 https://github.com/prometheus/prometheus/blob/main/LICENSE
+* Alert Manager: Apache 2.0 https://github.com/prometheus/alertmanager/blob/master/LICENSE
+* Jaeger: Apache 2.0 https://github.com/jaegertracing/jaeger/blob/master/LICENSE
+* Nginx: https://github.com/nginxinc/docker-nginx/blob/master/LICENSE
+* Prometheus Merge Tool: Apache 2.0 https://github.com/lablabs/prometheus-alert-overrider/blob/master/LICENSE
+* Alpine.js: MIT https://github.com/alpinejs/alpine/blob/main/LICENSE.md
+* Couchbase Cluster Monitor: Proprietary to Couchbase https://github.com/couchbaselabs/cbmultimanager/blob/master/LICENSE
+
+All licences are in the source repository and the microlith container in the link:https://github.com/couchbaselabs/observability/blob/main/microlith/licenses/[`/licenses`] directory.
+
+A statement is printed out to standard output/console at start up to indicate acceptance of licensing and where you can find them all.
+
+A simple link:https://github.com/couchbaselabs/observability/blob/main/tools/build-oss-container.sh[helper script] is provided as well to build without any proprietary components.
+
+== Container layers
+
+The various external components are reused directly from the layers already published by the entities responsible (e.g. Grafana publish the Loki image layers): this container is not redistributing or repackaging them but using as-is.
```

Comment:
Aren't we redistributing them though?

Reply:
If this is the wording Legal approves of then I'm happy, was just slightly difficult to parse.